profiles/kentik_snmp/apc/apc_ats.yml

```diff
@@ -0,0 +1,346 @@
+        sourceA: 1
+        sourceB: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.3
+      name: atsStatusRedundancyState
+      enum:
+        atsRedundancyLost: 1
+        atsFullyRedundant: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.4
+      name: atsStatusOverCurrentState
+      enum:
+        atsOverCurrent: 1
+        atsCurrentOK: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.5
+      name: atsStatus5VPowerSupply
+      enum:
+        atsPowerSupplyFailure: 1
+        atsPowerSupplyOK: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.6
+      name: atsStatus24VPowerSupply
+      enum:
+        atsPowerSupplyFailure: 1
+        atsPowerSupplyOK: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.7
+      name: atsStatus24VSourceBPowerSupply
+      enum:
+        atsPowerSupplyFailure: 1
+        atsPowerSupplyOK: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.8
+      name: atsStatusPlus12VPowerSupply
+      enum:
+        atsPowerSupplyFailure: 1
+        atsPowerSupplyOK: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.9
+      name: atsStatusMinus12VPowerSupply
+      enum:
+        atsPowerSupplyFailure: 1
+        atsPowerSupplyOK: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.10
+      name: atsStatusSwitchStatus
+      enum:
+        fail: 1
+        ok: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.11
+      name: atsStatusFrontPanel
+      enum:
+        locked: 1
+        unlocked: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.12
+      name: atsStatusSourceAStatus
+      enum:
+        fail: 1
+        ok: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.13
+      name: atsStatusSourceBStatus
+      enum:
+        fail: 1
+        ok: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.14
+      name: atsStatusPhaseSyncStatus
+      enum:
+        inSync: 1
+        outOfSync: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.15
+      name: atsStatusVoltageOutStatus
+      enum:
+        fail: 1
+        ok: 2
+  - MIB: PowerNet-MIB_ATS
+    symbol:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.1.16
+      name: atsStatusHardwareStatus
+      enum:
+        fail: 1
+        ok: 2
+
+  - MIB: PowerNet-MIB_ATS
+    table:
+      OID: 1.3.6.1.4.1.318.1.1.8.5.3.2
+      name: atsInputTable
+    symbols:
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.2.1.2
+        name: atsNumInputPhases
+      - OID: 1.3.6.1.4.1.318.1.1.8.5.3.2.1.3
+        name: atsInputVoltageOrientation
+        enum:
+          unknown: 1
+          singlePhase: 2
+          splitPhase: 3
+          threePhaseToNeutral: 4
+          threePhaseToPhase: 5
```

Comment:
these should be metric tags as they are non-volatile and more important as decorations to the `atsInputFrequency` OID

Reply:
Updated in latest commit.